en/nearest-neighbor-search-guide.md

```diff
@@ -262,7 +262,7 @@ schema track {
     }
 
     document-summary track_id {
-        summary track_id type string { 
+        summary track_id {
```

Comment:
I suggest removing the `source` specification as well to make the example as simple as possible.